test/src/helpers/hash/brcypt.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node, mocha */
+
+const assert = require('assert');
+const bcryptModule = require('../../../../src/helpers/hash/bcrypt');
+
+describe('bcrypt-hash', () => {
+  it('Should return false', async () => {
```

Comment:
It's a good practice to group the test within the same describe, if they are testing the same function,

```
describe('someFunction`, () => {
 it('test1', () => {});
 it('test2', () => {});
 it('test3', () => {});
});